iOS Client/Store/MBStore.swift

```diff
@@ -96,51 +118,47 @@ class MBStore: NSObject {
         performDownload(clientFunction: client.getArticlesWithCompletion, persistentContainer: persistentContainer, deserializeFunc: MBArticle.deserialize, completion: completion)
     }
     
-    private func loadDevotions(completion: @escaping ([MBDevotion]?, Error?) -> Void) {
-        
+    func saveDevotions(devotions: [LoadedDevotion]) throws {
+        try self.save(devotions, "devotions")
+    }
+    
+    private func read<T: Codable>(fromPath: String, _ resource: T.Type) throws -> T {
+        let (manager, path, _) = try urlPackage(forPath: "\(fromPath)")
```

Comment:
if fromPath is already a string, then why do string interpolation?

Reply:
Ah just left over from earlier we don't need to do string interpolation any more